certified-connectors/bitskout/apiDefinition.swagger.json

```diff
@@ -0,0 +1,176 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Bitskout",
```

Comment:
@saferoombiz Please fix the indentation. All files should be properly formatted.